data/tokens.json

```diff
@@ -3594,16 +3573,36 @@
         "value": "{gtmds_space-xl}*2",
         "type": "spacing"
       },
-      "gtmds_canvas-menu_margin": {
+      "gtmds_canvas-menu_padding": {
         "value": "{gtmds_size-md}",
         "type": "spacing"
       },
-      "gtmds_canvas-menu_padding": {
+      "gtmds_canvas-menu_margin": {
+        "value": "{gtmds_size-sm}",
+        "type": "spacing"
```

Comment:
I know initially, you included the padding in the component itself, why have you decided to remove them? Because of the whiteboard template?

Reply:
It is not removed, you can see it above via the offset tokens

Reply:
I meant for the component itself, the tokens are flawless